README.md

```diff
@@ -1,6 +1,6 @@
 # deltachat-desktop
 
-> Desktop Application for [delta.chat](https://delta.chat)
+**Desktop Application for [delta.chat](https://delta.chat)**
```

Comment:
Why this change?

Reply:
I assumed that this was an oversight (and to me it looks wrong). 

`>` is meant for quotes but that sentence isn't, so I replaced it with another form of emphasis that doesn't look like somebody accidentally formatted it as a quote.

Reply:
It's quite common to use `>` for short descriptions in README.

Reply:
But I'll let your change pass to keep _everyone_ happy but me.